nasa-gibs/worldview
# WV-2544 linkcheck

## Description
`npm run linkcheck` is a useful script which must be ran manually. It also takes a long time to run (~ 20 mins). This PR updates the shell script to run the command & sets the node-fetch package at 2.6.9 (newer versions have compatibility issues with bamboo). 

Fixes # WV-2544

## Note:
The node-fetch version here differs from the version in the `Updating Dependencies v4.3.0` branch. We will need to resolve that difference before pushing to develop.

## How To Test
This is already confirmed working in bamboo. A simple install/build/review should be sufficient.

@nasa-gibs/worldview
